common/src/models/resource.rs

```diff
@@ -95,57 +102,61 @@ fn get_secrets_table(secrets: &[&Response]) -> String {
     }
 
     format!(
-        r#"These secrets can be accessed by the service
+        r#"These secrets can be accessed by {}
 {table}
 "#,
+        service_name.bold()
     )
 }
 
-fn get_static_folder_table(static_folders: &[&Response]) -> String {
+fn get_static_folder_table(static_folders: &[&Response], service_name: &str) -> String {
     let mut table = Table::new();
 
     table
         .load_preset(UTF8_FULL)
         .apply_modifier(UTF8_ROUND_CORNERS)
-        .set_content_arrangement(ContentArrangement::DynamicFullWidth)
-        .set_header(vec![
-            Cell::new("Static Folders").set_alignment(CellAlignment::Center)
-        ]);
+        .set_header(vec![Cell::new("Static Folders")
+            .set_alignment(CellAlignment::Center)
+            .add_attribute(Attribute::Bold)]);
 
     for folder in static_folders {
         let path = serde_json::from_value::<PathBuf>(folder.data.clone())
             .unwrap()
-            .display()
-            .to_string();
+            .file_name()
+            .expect("static folder path should have a final component")
+            .to_str()
+            .expect("static folder file name should be valid unicode")
+            .to_owned();
```

Comment:
Can't we rather get it from `folder.config` somehow?

Reply:
Ah, I didn't think of that! Yes, that should be possible. I likely won't have time to address these today, by the way, but I can get to them in the morning.